src/main/webapp/i18n/de/lecture.json

```
@@ -97,15 +97,42 @@
                 "title": "Anhang",
                 "attachmentIDError": "Ungültiger Anhang oder ungültige Anhangs-ID.",
                 "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID.",
+                "noPDFAvailable": "Kein PDF verfügbar",
+                "cancelButton": "Abbruch",
                 "removePageButton": "Seiten entfernen",
+                "hidePageButton": "Seiten ausblenden",
+                "showPageButton": "Seiten anzeigen",
+                "showPagesTitle": "Seiten anzeigen",
                 "appendPDFButton": "PDF anhängen",
                 "attachmentUpdateSuccess": "Anhang erfolgreich aktualisiert.",
                 "attachmentUpdateError": "Fehler beim Aktualisieren des Anhangs: {{error}}",
                 "mergeFailedError": "Fehler beim Zusammenführen der Dateien: {{error}}",
                 "pageDeleteError": "Fehler beim Löschen der Seiten: {{error}}",
                 "fileSizeError": "Die Datei ist zu groß. Die maximale Dateigröße beträgt 20MB.",
                 "deletePagesQuestion": "Möchtest du die ausgewählten Seiten wirklich löschen?",
-                "deleteAllPagesQuestion": "Möchtest du wirklich alle Seiten löschen? Dies führt zur Löschung des Anhangs."
+                "deleteAllPagesQuestion": "Möchtest du wirklich alle Seiten löschen? Dies führt zur Löschung des Anhangs.",
+                "confirmCancelEditing": "Abbruchvorgang bestätigen",
+                "cancelEditingQuestion": "Möchtest du wirklich gehen, ohne deine Änderungen zu speichern?",
+                "invalidFileType": "Du kannst nur PDF-Dateien hochladen.",
+                "studentVersionCreateError": "Die Studentenversion der Dateieinheit konnte nicht erstellt werden.",
+                "studentVersionUpdateError": "Die Studentenversion der Dateieinheit konnte nicht aktualisiert werden..",
```

```suggestion
                "studentVersionCreateError": "Die Studierenden-Version der Datei konnte nicht erstellt werden.",
                "studentVersionUpdateError": "Die Studierenden-Version der Datei konnte nicht aktualisiert werden..",
```

according to our language guidelines: studierende = inclusive, studenten = (generic) male